docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_automanage/arc_automanage_machine_configuration_custom_linux/_index.md

```diff
@@ -0,0 +1,399 @@
+
+![Screenshot of PowerShell script](./23.png)
+
+Authenticate to Azure
+
+```powershell
+Connect-AzAccount -UseDeviceAuthentication
+
+# Update the values to reflect your ArcBox deployment
+$ResourceGroupName = "arcbox-demo-rg"
+$Location = "northeurope"
+```
+
+Create storage account for storing DSC artifacts
+
+```powershell
+$storageaccountsuffix = -join ((97..122) | Get-Random -Count 5 | % {[char]$_})
+
+New-AzStorageAccount -ResourceGroupName $ResourceGroupName -Name "arcboxmachineconfig$storageaccountsuffix" -SkuName 'Standard_LRS' -Location $Location -OutVariable storageaccount | New-AzStorageContainer -Name machineconfiguration -Permission Blob
+```
+
+> *Make a note of the storage account name, as this will be needed in later steps.*
+
+## Custom configuration for Linux
+
+The following steps needs to be executed from PowerShell 7 on the authoring machine for Linux.
+
+```powershell
+Import-Module PSDesiredStateConfiguration -RequiredVersion 3.0.0
+
+Configuration AzureArcJumpstart_Linux
+{
+
+    Import-DscResource -ModuleName nxtools -ModuleVersion 0.4.0
+
+    Node localhost
+    {
+
+      nxPackage nginx
+      {
+          Name = "nginx"
+          Ensure = "Present"
+      }
+      nxPackage hello
+      {
+          Name = "hello"
+          Ensure = "Present"
+      }
+<#
+
+    Due to a known issue with multiple class-based resources
+    we currently can`t leverage more than 1 DSC resource:
+    https://github.com/Azure/nxtools/issues/15
+
+      nxPackage powershell
+      {
+          Name = "powershell"
+          Version = "7.3.3"
+          Ensure = "Present"
+          #PackageType = "snap"
+      }
+
+      nxFile demo {
+        DestinationPath = "/tmp/arc-demo"
+        Ensure = "Present"
+        Contents = "Hello Arc!"
+        }
+
+      nxGroup arcusers {
+        GroupName = "arcusers"
+        Ensure = "Present"
+      }
+#>
+
+    }
+}
+
+$OutputPath = "$HOME/arc_automanage_machine_configuration_custom_linux"
+New-Item $OutputPath -Force -ItemType Directory
+
+AzureArcJumpstart_Linux -OutputPath $OutputPath
+```
+
+Create a package that will audit and apply the configuration (Set)
+
+```powershell
+New-GuestConfigurationPackage `
+-Name 'AzureArcJumpstart_Linux' `
+-Configuration "$OutputPath/localhost.mof" `
+-Type AuditAndSet `
+-Path $OutputPath `
+-Force
+```
+
+Test applying the configuration to the local machine
+
+```powershell
+Start-GuestConfigurationPackageRemediation -Path "$OutputPath/AzureArcJumpstart_Linux.zip"
+```
+
+Upload the configuration package to Azure Storage.
+Insert the correct storage account name on the first line in place of the placeholder value for the -Name parameter based on the output of the storage account created in the step *"Shared resources"*.
```

Comment:
For Italic, use "_" instead of "*"
```suggestion
Insert the correct storage account name on the first line in place of the placeholder value for the -Name parameter based on the output of the storage account created in the step _"Shared resources"_.
```